raiden/message_handler.py

```diff
@@ -100,23 +101,24 @@ def handle_message_secretrequest(raiden: RaidenService, message: SecretRequest,
             raiden.handle_and_track_state_change(secret_request)
 
     @staticmethod
-    def handle_message_revealsecret(raiden: RaidenService, message: RevealSecret, is_light_client=False) -> None:
+    def handle_message_revealsecret(raiden: RaidenService, message: RevealSecret, node_address: Address, is_light_client=False) -> None:
```

Comment:
and here.

also, i understand this is probably too much of a change to the codebase right now, but if we already have `message.sender`, it seems organic that the receiver is also part of the message variable.

Reply:
Yes, but the receiver isnt on each Message of the protocol and we wanted to stick to the raiden format of the msgs for now

Reply:
Done here